src/graphql/queries.js

```diff
@@ -438,17 +438,16 @@ module.exports = {
     if (voterFilters.length !== 1) {
       throw Error('only one event is allowed');
     }
-    if (voterFilters[0].topicAddress == null) {
```

Comment:
why remove the check?

Reply:
since when we do global leaderboard, we don't need specific topicAddress